requirements.txt

```diff
@@ -7,7 +7,7 @@ Ipython>=7.16.3
 ipywidgets>=7.5.1
 jinja2>=2.10
 jsonpatch>=1.22
-jsonschema>=2.5.1
+jsonschema>=2.5.1,<=4.7.2
```

Comment:
I had to set the max jsonschema pkg version because, with latest versions the error message from validate() is not the one that is being tested [here](https://github.com/great-expectations/great_expectations/blob/0913b88c1e24117528acad3cf4f25ce8d80d4b14/tests/core/usage_statistics/test_schema_validation.py#L311): 
`assert "'variable_count' is a required property" in str(e.value)`.

I only got this problem with this test. I see two choices:
1. Upd the message in the test assertion and do not set the max pkg version.
2. Set the max pkg version and keep it as is.

I followed the second one.